src/main/java/uk/gov/beis/els/model/GoogleAnalyticsEventParams.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.beis.els.model;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class GoogleAnalyticsEventParams {
+  private final Map<String, String> params;
+
+  public GoogleAnalyticsEventParams() {
+    params = new HashMap<>();
+  }
+
+  public void addParam(String name, String value) throws RuntimeException {

```

Comment:
Nit: `RuntimeException` is unchecked so convention is not to declare it on the method